lib/single-new-eks-fargate-opensource-observability-pattern/index.ts

```diff
@@ -0,0 +1,119 @@
+import { Construct } from 'constructs';
```

Comment:
This file should be moved under `single-new-eks-opensource-observability-pattern`  so you can avoid the extra file for `grafanaoperatorsecretaddon.ts`